modules/commons/src/main/resources/schemas/neurosciencegraph/commons/protocol/v0.1.5.json

```diff
@@ -0,0 +1,69 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/commons/protocol/v0.1.4/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0",
+    "{{base}}/schemas/nexus/schemaorgsh/identifier/v0.1.0",
```

Comment:
Import the latest commons/identifier/ schema

Reply:
I think this might be the latest already? https://github.com/INCF/neuroshapes/blob/master/modules/nexus-schemas/src/main/resources/schemas/nexus/schemaorgsh/identifier/v0.1.0.json

Reply:
There is a [v0.2.0](https://github.com/INCF/neuroshapes/blob/master/modules/commons/src/main/resources/schemas/neurosciencegraph/commons/identifier/v0.2.0.json) version.